openfisca-web-api/README.md

```diff
@@ -53,3 +30,10 @@ Computations done by <a href="https://openfisca.fr">OpenFisca</a>, the <a href="
 ```html
 Calculs effectués par <a href="https://openfisca.fr">OpenFisca</a>, le moteur <a href="https://choosealicense.com/licenses/agpl-3.0/" title="AGPL-3.0">libre et ouvert</a> du système social et fiscal. Code source disponible sur <a href="https://github.com/openfisca">github.com/openfisca</a>.
 ```
+## Hosting an API instance
+
+To allow app developers to access your country package information and computation, you [deploy your own instance](https://github.com/openfisca/openfisca-core#serving-the-api).
+
+### Track your API
+
+If you want to track how your API is being used, you can install the [OpenFisca Tracker](https://github.com/openfisca/tracker)
```

Comment:
> Set link to `https://github.com/openfisca/openfisca-core#tracker` instead?
If the user clicked on the previous link he might see the tracker section of the core readme and be surprised by this other link?

Add a point at end of line?